.eslintrc.js

```diff
@@ -3,6 +3,7 @@ const { getBaseConfig } = require('@openedx/frontend-build');
 
 const config = getBaseConfig('eslint');
 
+config.globals = { globalThis: 'readonly' };
```

Comment:
@bradenmacdonald [curious] Lots of repos use `global` today; I'm wondering if we should update the base ESLint config defined in `@openedx/frontend-build` so each consuming shared JS library and/or MFE doesn't need to specify this config across many different repos. Curious your two cents.

On a similar note, I wonder if it'd be more acceptable to extend the shared ESLint config's `env` option to include `es2020` so that other similar syntax is deemed acceptable as well? `@openedx/frontend-build` relies on the `env` defined in `@edx/eslint-config` [here](https://github.com/openedx/eslint-config/blob/89d0227e2d1569b7b9cdf8a29e37fbb3e5600c12/.eslintrc.js#L20-L24). Setting `es2020` in `env` would in place of configuring the `globals` for `globalThis` explicitly.

Reply:
Yes, I'm definitely in favor of setting `es2020` in `env` as a default!

Reply:
@bradenmacdonald Opened a PR to the base shared ESLint configuration (`@edx/eslint-config`): https://github.com/openedx/eslint-config/pull/153

I believe having `es2020` in there would eliminate the need to keep `es6`, since the environments (as far as I can tell) are cumulative/additive, where a later environment supersedes an earlier one. That said, I'm opting to keep `es6` in there for now just in case that is not true.

Let me know your thoughts; will also share in FWG Slack :)